File: M-Scout 15/M-Scout 15/TeamWebsite.m
//
//  TeamWebsite.m
//  M-Scout 15
//
//  Created by Brendan Boyle on 2/23/15.
//  Copyright (c) 2015 FRC: Team 3926. All rights reserved.
//

#import "TeamWebsite.h"
#import "SVWebViewController.h"

@interface TeamWebsite ()

@end

@implementation TeamWebsite

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    SVWebViewController *webViewController = [[SVWebViewController alloc] initWithAddress:@"http://team3926.org"];
    [self.navigationController pushViewController:webViewController animated:NO];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

/*
#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    // Get the new view controller using [segue destinationViewController].
    // Pass the selected object to the new view controller.
}
*/

@end

File: M-Scout 15/M-Scout 15/TeamWebsite.swift
//
//  TeamWebsite.swift
//  M-Scout 15
//
//  Created by Brendan Boyle on 2/23/15.
//  Copyright (c) 2015 FRC: Team 3926. All rights reserved.
//

import UIKit
import WebKit

class TeamWebsite: UIViewController {

    @IBOutlet var containerView : UIView! = nil
    var webView: WKWebView?
    
    override func loadView() {
        super.loadView()
        
        self.webView = WKWebView()
        self.view = self.webView!
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        var url = NSURL(string:"http://www.team3926.org/")
        var req = NSURLRequest(URL:url!)
        self.webView!.loadRequest(req)
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }

}
